frontend/src/pages/NewPlacePage.vue

```diff
@@ -13,7 +13,12 @@
                                 <div class="field-body">
                                     <div class="field">
                                         <div class="control is-expanded">
```

Comment:
NewPlacePage.vue is wrapper component. Create `AddPlaceForm.vue` and move code there

Reply:
Done.